src/javascripts/components/airportList/airportList.js

```diff
@@ -40,25 +41,46 @@ const addAirportEvent = (e) => {
       utils.printToDom('#new-airport', '');
     })
     .catch((err) => console.error('could not add airport', err));
-
-  console.warn(newAirport);
 };
 
 const removeAirportEvent = (e) => {
   const airportId = e.target.closest('.airport-card').id;
-  console.warn(airportId);
   airportData.deleteAirport(airportId)
     .then((response) => {
       console.warn('response', response);
-
       buildHangar();
     })
     .catch((err) => console.error('did not delete airport', err));
 };
 
+const editAirportEvent = (e) => {
+  e.preventDefault();
+
+  const airportId = e.target.closest('.edit-airport').id;
+
+  const editedAirport = {
+    imgURL: $('#edit-airport-pic').val(),
+    name: $('#edit-airport-name').val(),
+    location: $('#edit-airport-location').val(),
+    webURL: $('#edit-airport-website').val(),
+  };
+  airportData.updateAirport(airportId, editedAirport)
+    .then(() => {
+      buildHangar();
+      utils.printToDom('#edit-airport', '');
+    })
+    .catch((err) => console.error('could not edit airport', err));
+};
+
+const showAirportForm = (e) => {
+  editAirport.showForm(e.target.closest('.airport-card').id);
+};
+
 const airportEvents = () => {
   $('body').on('click', '.delete-airport', removeAirportEvent);
+  $('body').on('click', '.edit-airport', showAirportForm);
   $('body').on('click', '#airport-creator', addAirportEvent);
+  $('body').on('click', '', editAirportEvent);
```

Comment:
ahhh, I betcha this is the issue. You're saying that we should run the `editAirportEvent` no matter what gets clicked on in the body, so I betcha `airportId` isn't set to anything, so when you do the update it wipes everything out 🤔  just a guess though. Try making sure this is targeting the correct element